SU-SWS/ood-giving-campaign
# GIVCAMP-359 | Mobile story list nav

# READY FOR REVIEW

# Summary
- Update headless ui
- Mobile functionality and styling of story list page nav

# Review By (Date)
- Retro

# Criticality
- 7

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to storyblok and go to the /stories page. Select PR 302 from the visual editor.
2. Pop the preview into its own window and copy the URL
3. Test this URL on your mobile device
4. The filter menu should look like this on mobile:
![Screenshot 2024-06-17 at 2 49 14 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/808efbc6-caba-4036-bd6b-cf1c8453522d)
5. Click the button and open up the filter modal
NOTE: you can't click the menu items now because the pages are not published, but I've tested this by temporarily publishing a few pages myself with noIndex=true
6. Check that you can close the modal by clicking the x button


# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-302
